services/provider/server/server.go

```diff
@@ -497,9 +497,15 @@ func (s *OCSProviderServer) FulfillStorageClassClaim(ctx context.Context, req *p
 
 	klog.Infof("Found StorageConsumer %q (%q)", consumerObj.Name, req.StorageConsumerUUID)
 
-	err = s.storageClassClaimManager.Create(ctx, consumerObj, req.StorageClassClaimName, req.StorageType.String(), req.EncryptionMethod)
+	storageType := "sharedfilesystem"
+	if req.StorageType == pb.FulfillStorageClassClaimRequest_BLOCKPOOL {
+		storageType = "blockpool"
+	}
```

Comment:
Let's cover each option individually and cover the case of unknown values

```suggestion
	var storageType string
	switch req.StorageType {
	case pb.FulfillStorageClassClaimRequest_BLOCKPOOL:
		storageType = "blockpool"
	case FulfillStorageClassClaimRequest_SHAREDFILESYSTEM:
			storageType := "sharedfilesystem"
	default:
		return fmt.ErrorF("Encountered an unknown stroage type, %s", storageType) 
	
```

Reply:
Done.